src/index.html

```diff
@@ -16,22 +16,18 @@
   <meta name="apple-mobile-web-app-capable" content="yes">
 
   <script>
-  var config = {
-    apiEndpoint: 'http://localhost:8000',
-    passageEndpoint: 'http://localhost:5001',
-
-    // environment is set to:
-    // 'dev' while developing
-    // 'kubernetes' when deployed in production
-    // 'docker-container' when the final build container is run manually
-    // the behaviour is configured by the ENVIRONMENT variable,
-    // see scripts/start.sh
-    environment: 'development',
-    ingressBaseDomain: 'k8s.sample.io',
-    awsCapabilitiesJSON: '{"m3.large":{"description":"M3 General Purpose Large","memory_size_gb":7.5,"cpu_cores":2,"storage_size_gb":32},"m3.xlarge":{"description":"M3 General Purpose Extra Large","memory_size_gb":15,"cpu_cores":4,"storage_size_gb":80},"m3.2xlarge":{"description":"M3 General Purpose Double Extra Large","memory_size_gb":30,"cpu_cores":8,"storage_size_gb":160}, "m4.xlarge":{"cpu_cores":4,"description":"M4 General Purpose Extra Large","memory_size_gb":16,"storage_size_gb":0}}',
-    azureCapabilitiesJSON: '{"Standard_A2_v2":{"additionalProperties":{},"description":"This is some description","maxDataDiskCount":4,"memoryInMb":4294.967296,"name":"Standard_A2_v2","numberOfCores":2,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":21474.83648},"Standard_A4_v2":{"additionalProperties":{},"description":"Here is a longer description that might be too long for the field","maxDataDiskCount":8,"memoryInMb":8589.934592,"name":"Standard_A4_v2","numberOfCores":4,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":42949.67296},"Standard_A8_v2":{"additionalProperties":{},"description":"Another VM size description text","maxDataDiskCount":16,"memoryInMb":17179.869184,"name":"Standard_A8_v2","numberOfCores":8,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":85899.34592}, "Standard_D2s_v3":{"additionalProperties":{},"description":"Dsv3-series, general purpose, 160-190 ACU, premium storage supported","maxDataDiskCount":4,"memoryInMb":8589.935,"name":"Standard_D2s_v3","numberOfCores":2,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":17179.87}}',
-    happaVersion: 'development',
-  };
+var config = {
+  // apiEndpoint: "https://api.g8s.seal.us-east-1.aws.gigantic.io",
+  // apiEndpoint: "https://api.g8s.gauss.eu-central-1.aws.gigantic.io",
+  apiEndpoint: "https://gs-api.g8s.eu-central-1.aws.cps.vodafone.com",
```

Comment:
Don't forget these in 😬

Reply:
already changed 🤦‍♂  forgot to push, doing it now